[CUBRIDMAN-298] [EN] [KO] 매뉴얼 구조 정리와 프로시저 등록에 대한 설명 강화 (#643)
CUBRID/cubrid-manual: en/pl/jsp.rst

> :meta-description: CUBRID supports to develop stored functions and procedures in Java. Java stored functions/procedures are executed on the JVM (Java Virtual Machine) hosted by CUBRID.

Reviewer: ```suggestion
:meta-description: CUBRID supports the development of stored functions and procedures in Java. Java stored functions/procedures are executed on the JVM (Java Virtual Machine) hosted by CUBRID.
```

******************************
Java Stored Procedure
******************************

CUBRID supports the development of stored functions and procedures using Java. Java stored functions and procedures are executed on a Java Virtual Machine (JVM) hosted by CUBRID.
Since Java stored functions and procedures use the JVM, they offer excellent interoperability and portability.

.. _jsp-introduction:

Introduction to Java Stored Procedure
==============================================

> CUBRID supports to develop stored functions and procedures in Java. Java stored functions/procedures are executed on the JVM (Java Virtual Machine) hosted by CUBRID.

Reviewer: ```suggestion
CUBRID supports the development of stored functions and procedures in Java. Java stored functions/procedures are executed on the JVM (Java Virtual Machine) hosted by CUBRID.
```
